projects/client/RabbitMQ.Client/src/client/api/ConnectionFactory.cs

```diff
@@ -141,6 +142,18 @@ public class ConnectionFactory : ConnectionFactoryBase, IAsyncConnectionFactory
         /// <remarks> PLEASE KEEP THIS MATCHING THE DOC ABOVE.</remarks>
         public const string DefaultVHost = "/";
 
+        /// <summary>
+        /// The default AMQP URI SSL protocols.
+        /// </summary>
+        public static SslProtocols DefaultAmqpUriSslProtocols { get; set; } =
```

Comment:
Why does this have a setter? If that's the default, shouldn't it be get-only?

Reply:
In case the user want's to change the default globally.

Reply:
For this particular property it sounds OK to me to allow for overriding.

Reply:
`AmqpUriSslProtocols` would actually be the one overriding the default.

Reply:
Then I agree with @bording: the setter can be dropped. We don't need two ways to override the default.

Reply:
The intention was to have a wider default that would apply to all connection factories if not changed and a setting for each connection factory.

Reply:
Fair enough, the setter is something we can accept.